docs/guides/integrating-with-contentful.md

```diff
@@ -68,13 +68,14 @@ await init({
 });
 
 let userid = “user id”
-let flag_key = <EPPO FLAG KEY>
+let flagKey = <EPPO FLAG KEY>
```

Comment:
JS does not use snake_case

Reply:
Should we also write `userId`?

Reply:
while in here, these should be `const` and terminate with semicolons 😛 

And maybe make it valid by putting the placeholder inside a string.
```ts
const flagKey = "<EPPO FLAG KEY>";
```

Reply:
Good catch @aarsilv ; cleaned this up a bit more.